pv_site_api/main.py

```diff
@@ -276,7 +277,26 @@ def post_pv_actual(
             }
         )
 
+    # Set the error generation capacity factor from environment variable
+    capacity_factor = float(os.getenv("ERROR_GENERATION_CAPACITY_FACTOR", 1.1))
+
     generation_values_df = pd.DataFrame(generations)
+    site = get_site_by_uuid(session=session, site_uuid=site_uuid)
+    site_capacity_kw = site.capacity_kw
+    exceeded_capacity = generation_values_df[
+        generation_values_df["power_kw"] > site_capacity_kw * capacity_factor
+    ]
+    if len(exceeded_capacity) > 0:
+        raise HTTPException(
+            status_code=422,
+            detail=(
+                "Error processing generation values. "
```

Comment:
Oh and I think we need a f-string here, so itll be `f"`. This makes the variables get resolves. Also on another couple of strings in this block